README.md

```diff
@@ -1,2 +1,437 @@
+  ```js
+  // good
+  cars.filter((car) => car.color == 'red');
+
+  // bad
+  cars.filter(car => car.color == 'red');
+  ```
+
+* Object whitespace
+  ```js
+  // good
+  const car = {color: 'red', model: 'Saturn'};
+
+  // bad
+  const car = { color: 'red', model: 'Saturn' };
+  // bad
+  const car = {color:'red',model:'Saturn'};
+  ```
+
+
+## Common packages
+
+## Directories and files
+
+### Directory structure
+
+### Filenames
+
+* Use camelCase that matches the default export (this makes it easier to edit in a text editor)
+
+## Imports
+
+* Import modules in order of more general to more specific:
+  - Utility libraries (e.g. lodash)
+  - Frameworks (e.g. Redux)
+  - Utility functions/constants
+  - Actions
+  - Components
+
+* When there are more than 5 import statements, add blank lines between groups.
+
+  ```js
+  import { connect } from 'react-redux';
+  import { Link } from 'react-router-dom';
+  import { Card, Button } from 'semantic-ui-react';
+  import { fetchUsers } from 'actions/users';
+  import searchBy from 'utils/search';
+
+  import Search from 'components/helpers/Search';
+  import Page from 'components/layouts/Page';
+  ```
+
+## JSX
+
+JSX leverages syntactic similarity with HTML so similar conventions should be observed.
+
+* rendering
+  ```js
+  // good
+  render() {
+    return <Button text="Save" onChange={this.save}/>;
+  }
+  // good
+  render() {
+    return (
+      <div className="action-bar">
+        <Button text="Save" onChange={this.save}/>
+        <Button text="Cancel" onChange={this.goBack}/>
+      </div>
+    );
+  }
+
+  // bad
+  render() {
+    return <div className="action-bar">
+      <Button text="Save" onChange={this.save}/>
+      <Button text="Cancel" onChange={this.goBack}/>
+    </div>;
+  }
+  ```
+
+* use double quotes for attributes
+  ```js
+  // good
+  <Button text="Save" onChange={this.save}/>
+
+  // bad
+  <Button text='Save' onChange={this.save}/>
+  ```
+
+* Do not use braces around double quotes
+  ```js
+  // good
+  <Button text="Save" onChange={this.save}/>
+
+  // bad
+  <Button text={"Save"} onChange={this.save}/>
+  ```
+
+* Do not use spaces around =
+  ```js
+  // good
+  <Button text="Save" onChange={this.save}/>
+
+  // bad
+  <Button text = "Save" onChange = {this.save}/>
+  ```
+
+* Inline short components
+  ```js
+  // good
+  <Button text="Save" onChange={this.save}/>
+
+  // bad
+  <Button
+    text="Save"
+    onChange={this.save}/>
+  ```
+
+* Multiline each prop on it's own line
+  ```js
+  // good
+  <Input
+    type="email"
+    label="Email"
+    placeholder="name@example.com"
+    onChange={this.setValue('email')}
+    onClick={this.selectText}/>
+
+  // bad
+  <Input
+    type="email" label="Email" placeholder="name@example.com"
+    onChange={this.setValue('email')} onClick={this.selectText}/>
+  ```
+
+* Keys should come first and may be on the first line when multilined.
+  ```js
+  // ok
```

Comment:
good instead of ok for consistency?

Reply:
maybe... I'm not sure if this should be a recommendation or suggestion

Reply:
I also prefer the consistency of "good"